hasura.planx.uk/migrations/1700303840464_setup_analytics_summary_view/up.sql

```diff
@@ -0,0 +1,21 @@
+CREATE OR REPLACE VIEW public.analytics_summary AS 
+select 
+	a.id as analytics_id,
+	al.id as analytics_log_id,
+	f.slug as service_slug,
+	t.slug as team_slug,
+	a.type as analytics_type,
+	a.created_at as analytics_created_at,
+	a.user_agent as user_agent,
```

Comment:
I'm hoping we can keep this as a JSONB column with the changes in https://github.com/theopensystemslab/planx-new/pull/2451

If not, I can just pull out the key bits of data here like browser, platform etc

Reply:
Agree with initially keeping as JSONB! 

nit: `a.user_agent as user_agent` is redundant / unnecessary and adds a bit of noise? only need `as` if actually renaming, table reference won't be visible in final view (applies to a number of columns below too)

Reply:
Good catch 👍